docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.5"
+
+services:
+  validate:
+    image: renovate/renovate:19.19.1-slim@sha256:6b35b7505ef8d7200671e124fe810b5e75d874cd5e366f7b139a75a26827655d
+    environment:
+      RENOVATE_CONFIG_FILE: /usr/src/renovate-config/renovate.json
```

Comment:
I used `/usr/src/renovate-config` because Renovate's source code is in `/usr/src/app`. So both use `/usr/src` as parent directory.

Also I would say that Docker mounts shouldn't always mount to `/mnt` because it's not the same like mounting an external volume.